src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -888,16 +888,45 @@ TEST_F(MessageHelperTest,
   }
 }
 
-TEST_F(MessageHelperTest, SubscribeApplicationToSoftButton_CallFromApp) {
-  // Create application mock
+TEST_F(MessageHelperTest,
+       SubscribeApplicationToSoftButton_SoftbuttonsIsAnsent_DoesntCallFromApp) {
```

Comment:
```suggestion
       SubscribeApplicationToSoftButton_SoftbuttonsAreAbsent_DoesntCallFromApp) {
```

Reply:
@ShobhitAd fixed in https://github.com/smartdevicelink/sdl_core/pull/2629/commits/6ef9cfb61065b10785ad4fc37f0b0b0463f86fb7